src/Microsoft.Health.Fhir.Tests.Common/Categories.cs

```diff
@@ -30,5 +30,9 @@ public static class Categories
         public const string ExportDataValidation = "ExportDataValidation";
 
         public const string SmartOnFhir = "SmartOnFhir";
+
+        public const string DataConvert = "DataConvert";
+
+        public const string CustomDataConvert = "CustomDataConvert";
```

Comment:
>public const string CustomDataConvert = "CustomDataConvert"; [](start = 8, length = 60)

 No references found to 'Categories.CustomDataConvert' D:\src\fhir-server\src\Microsoft.Health.Fhir.Tests.Common\Categories.cs   Microsoft.Health.Fhir.Tests.Common    
Is it coming in future PRs?

Reply:
My mistake. It should be added to CustomDataConvert.cs. Fixed, thanks!